docs/current/guides/snippets/work-with-host-filesystem/glob-pattern/index.mts

```diff
@@ -0,0 +1,33 @@
+import { connect, Client } from "@dagger.io/dagger"
+import * as fs from "fs"
+import * as os from "os"
+import * as path from "path"
+
+const workdir = os.tmpdir()
+let folder = workdir + path.sep
+
+for (const subdir of ["foo", "bar", "baz"]) {
+  folder = path.join(folder, subdir)
+  fs.mkdirSync(folder)
+
+  for (const file of [".txt", ".out", ".rar"]) {
+    fs.writeFileSync(path.join(folder, subdir + file), subdir)
+  }
+}
+
+connect(
+  async (client: Client) => {
+    const daggerdir = await client.host().directory(workdir, {
+      include: ["**/*.rar", "**/*.txt"],
+      exclude: ["**.out"],
+    })
+
+    folder = "." + path.sep
+    for (const dir of ["foo", "bar", "baz"]) {
+      folder = path.join(folder, dir)
+      const entries = await daggerdir.entries({ path: folder })
+      console.log(entries)
+    }
+  },
+  { LogOutput: process.stderr, Workdir: workdir }
```

Comment:
This is linked to the ongoing discussion in https://discord.com/channels/707636530424053791/1145698713331834990/1146770191120875580 and the PR https://github.com/dagger/dagger/pull/5725/ which makes changes to the existing examples. I recommend waiting until that discussion is closed and PR is merged and then updating these examples.